main.py

```diff
@@ -285,7 +284,7 @@ def start(self, settings):
 
 		print('Running: %s' % self.launchCmdToString(cmd))
 
-		self._p = subprocess.Popen(cmd)
+		self._p.start(self.launchCmdToString(cmd))
```

Comment:
This launch-string is generated in the previous line already. Should be cached

Reply:
Done.